### PR TITLE
Reversion of a series

### PR DESCRIPTION
**Pull request by shivamvats**
@certik @thilinarmtb This is up for review.

### CODE REVIEW

File: sympy/polys/tests/test_ring_series.py [hunk 1]
```diff
@@ -99,6 +99,16 @@ def test2(p):
...
     raises(NotImplementedError, lambda: test2(p))
 
+def tet_series_reversion():
+    R, x, y = ring('x, y', QQ)
+
+    p = rs_tan(x, x, 10)
+    assert rs_series_inversion(p, x, 8, y) == rs_atan(y, y, 8)
+
+    p = rs_sin(x, x, 10)
+    assert rs_series_inversion(p, x, 8, y) == 5*y**7/112 + 3*y**5/40 + \
```
Comment (certik) [inline sympy/polys/tests/test_ring_series.py]: Why not to test `rs_asin`, similarly to how you test `rs_atan`?

**Comment by certik** [comment]
This looks very good. +1 to merge.

File: sympy/polys/tests/test_ring_series.py (hunk 1 above)
Comment (shivamvats) [inline sympy/polys/tests/test_ring_series.py]: @certik `asin` is in #9495, not yet merged.

Comment (certik) [inline sympy/polys/tests/test_ring_series.py]: I see. We can update this after it gets merged.

**Comment by shivamvats** [comment]
@certik Can we merge this?

**Comment by certik** [comment]
Yes. Thanks!

**Pull request merged**